src/Sentry/Platforms/Android/SentrySdk.cs

```diff
@@ -87,6 +90,7 @@ private static void InitSentryAndroidSdk(SentryOptions options)
             o.ServerName = options.ServerName;
             o.SessionTrackingIntervalMillis = (long)options.AutoSessionTrackingInterval.TotalMilliseconds;
             o.ShutdownTimeoutMillis = (long)options.ShutdownTimeout.TotalMilliseconds;
+            o.SetNativeHandlerStrategy(handlerStrategy ?? JavaSdk.Android.Core.NdkHandlerStrategy.SentryHandlerStrategyDefault);
```

Comment:
Why do we need the null coalesce operator if `ToJava` doesn't return null?

Besides complexity an additional issue with this is that now we have 2 places dealing with null and defining a default. Had we even needed to change the default we'd need to 'remember' to change these two spots